lib/folio_sync/folio/reader.rb

```diff
@@ -0,0 +1,15 @@
+module FolioSync
+  module Folio
+    class Reader
```

Comment:
If reading MARC from FOLIO ends up only involving a one-line call to `client.find_marc_record(instance_record_hrid: hrid)`, we might end up wanting to retire this wrapper class later, but I have no objection to keeping it for now while we're still figuring things out!

Reply:
Good point - we'll revisit if this class is still needed once we're closer to wrapping up the project